### PR TITLE
Implement `displayTickets` on iOS and Android

### PR DESCRIPTION
**Pull request by theVinesh**
- Bumps Android sdk to 15.2.0 and iOS sdk to 15.2.1
- Implements `displayTickets`

### CODE REVIEW

**Comment by theVinesh** [comment]
Still requires bumping `intercom_flutter_web`. I'll do it once that PR is merged

File: intercom_flutter/pubspec.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 homepage: https://github.com/v3rm0n/intercom_flutter
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  intercom_flutter_platform_interface: ^1.3.1
+  intercom_flutter_platform_interface: ^1.3.2
   intercom_flutter_web: ^0.3.2
```
Comment (deepak786) [inline intercom_flutter/pubspec.yaml]: web 0.3.3 version is published so please update the same.

Comment (theVinesh) [reply intercom_flutter/pubspec.yaml]: Thanks. It's now done @deepak786!

**Comment by theVinesh** [comment]
There seems to be some problem downloading the chrome driver on the ci

**Review by deepak786 — APPROVED** [review]

**Comment by deepak786** [comment]
> There seems to be some problem downloading the chrome driver on the ci

Yeah I will check.

**Pull request merged**